rfcs/text/0015/elf.yml

```
@@ -89,6 +89,12 @@
       description: >
         "0x1" for original ELF files.
 
+    - name: packers
+      type: keyword
+      level: extended
+      description: >
+        Packers used for the ELF file.
```

```suggestion
        Packers used for the ELF file.
     normalize:
       - array
     example: '[ "upx" ]'
```

Let's specify `packers` is intended to be a list.